### ISSUE TITLE
Fix consistency of ETag generation in example SDPs

### ISSUE THREAD

**Issue by cameronelliott** [wish-wg/webrtc-http-ingest-protocol#84]
A fix is proposed in #83 

While HTTP ETag should be opaque per RFC9110,
in this RFC there is a convention of generating ETag like so: Etag = offer-ice-ufrag + ':' + answer-ice-ufrag

Except, the values used in the first example are inconsistent and confusing.

It might be argued that since ETag should be opaque, this inconsistency doesn't matter.
But I find it confusing, and thinks others might also. (In the patch examples, you can see how the ETag is generated on an Ice restart)

It might also benefit the RFC to explain both:

The convention use for generating ETag
Why the convention isn't required (or is required) in the WHIP RFC

**Comment by murillo128** [comment]
As you say the Etag value is opaque, it could be implemented as that or with an incremental counter or with a timestamp one.

While I am ok with fixing the examples, i am against of adding the convention to the rfc as I don't want the clients to make any assumption of the value of the etag returned by the server.

**Comment by murillo128** [comment]
After double thinking about this, i think it is best to use a random value for the Etag in the examples so client implementations don't take it for granted

**Comment by cameronelliott** [comment]
I agree, I was confused by the not-opaque etags used in the examples, I support using opaque examples. Should be helpful for implementors.
